SharedPackages/BrowserServicesKit/Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -167,47 +167,50 @@ public final class NetworkProtectionKeychainTokenStore: NetworkProtectionTokenSt
 /// Writing a new auth token will replace the old one.
 public final class NetworkProtectionKeychainTokenStoreV2: AuthTokenStoring {
     private let keychainStore: NetworkProtectionKeychainStore
-    private let errorEvents: EventMapping<NetworkProtectionError>?
+    private let errorPixelHandler: EventMapping<NetworkProtectionError>?
 
     public struct Defaults {
+        static let bundleID = Bundle.main.bundleIdentifier ?? "com.duckduckgo.networkprotection"
         static let tokenStoreEntryLabel = "DuckDuckGo Network Protection Auth Token Container"
-        public static let tokenStoreService = "com.duckduckgo.networkprotection.authTokenContainer"
-        static let tokenStoreName = "com.duckduckgo.networkprotection.tokenContainer"
+        public static let tokenStoreService = "\(bundleID).authTokenContainer"
+        static let tokenStoreName = "\(bundleID).tokenContainer"
```

Comment:
Risk mitigation here is that authv2 is only internal, so any VPN disruption after the update will be only internal.

Reply:
yes, but only for macOS vpn and only for internal users that already migrated to authv2